dotnet/src/Connectors/Connectors.UnitTests/Memory/VectorStoreRecordPropertyReaderTests.cs

```diff
@@ -0,0 +1,259 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Memory;
+using Xunit;
+
+namespace SemanticKernel.Connectors.UnitTests.Memory;
+
+public class VectorStoreRecordPropertyReaderTests
```

Comment:
nit: Since the actual class is located in `dotnet/src/InternalUtilities/src/Schema`, do we want to put tests for it in `dotnet/src/SemanticKernel.UnitTests/Utilities`?

Reply:
Sure, I can move it.

Reply:
Moved, thanks.